packages/cli-tool/test/commands/generate/index.test.ts

```diff
@@ -1,17 +1,111 @@
+import * as fs from 'fs';
+
 import { expect, test } from '@oclif/test';
+import Inquirer from 'inquirer';
+
+import { bootstrapTestData } from '../../add-ons/ui-framework/bootstrap';
+import { tailwindCssTestData } from '../../add-ons/ui-framework/tailwind-css';
+import { gitHubTestData, gitLabTestData } from '../../add-ons/version-control';
+import { TestScenario } from '../../helpers/test-scenario';
+
+const templateRepoPath = 'file:./packages/cra-template';
+const projectName = 'test-app';
+const testFolderPath = '../../';
+
+const projectPath = `${testFolderPath}${projectName}`;
+
+const testScenarios: TestScenario[] = [
+  {
+    options: {
+      versionControl: 'github',
+      uiFramework: 'bootstrap',
+    },
+    testData: {
+      filesShouldExist: [
+        ...gitHubTestData.filesShouldExist,
```

Comment:
like the usage of spread operator a lot here, really makes this file cleaner, Thank you @malparty !